Corona-Warn-App/src/test/java/de/rki/coronawarnapp/ccl/configuration/update/CclConfigurationUpdaterTest.kt

```diff
@@ -73,46 +77,63 @@ internal class CclConfigurationUpdaterTest : BaseTest() {
         }
 
     @Test
-    fun `updateConfiguration() should return true when new booster rules or new configuration was downloaded or false otherwise`() =
+    fun `updateConfiguration() should return true if any item was downloaded or false otherwise`() =
```

Comment:
I think test above called `updateIfRequired() should NOT update booster rules and ccl configuration if NOT required but should trigger DccWalletInfo recalculation` should also verify that the `dccValidationRepository` was not called.